src/arrayMethodSort.js

```diff
@@ -3,9 +3,21 @@
 /**
  * Implement method Sort
  */
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction = (a, b) => `${a}` > `${b}`) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        if (compareFunction(this[i], this[j]) > 0) {
```

Comment:
The comparison function should return a number, but there's no guarantee that the result of the comparison here will be a number. Make sure to convert the result to a number to avoid any unexpected behavior.